c_common/front_end_common_lib/src/data_specification.c

```diff
@@ -25,15 +26,15 @@ typedef enum region_elements{
 //!        loaded into the user0 register of the core during the tool chain
 //!        loading.
 //! \return the SDRAM start address for this core.
-address_t data_specification_get_data_address() {
-
+address_t data_specification_get_data_address(void)
+{
```

Comment:
See previous messages re: inline brackets

Reply:
OK, but that isn't the style in the rest of the code (not in most of it anyway)...